scripts/ananse

```diff
@@ -332,6 +332,14 @@ if __name__ == "__main__":
         help="column of GRN file sorted to select top interactions, 'prob' by default",
         default='prob',
     )
+    group.add_argument(
+        "-e",
+        "--GRNwb-edges",
+        dest="GRN_wb",
+        help="set to true to plot weighted binding as the GRN edges in the GRN plot instead of the default interaction score",
+        action="store_true",
+        default=False,
```

Comment:
Plotting only takes a minute right? I think it should be `True` by default!

Reply:
I guess perhaps you understood it by your latter remark, but this is about the type of edge that gets plotted. I'll change it to a weight/wb string instead to stop confusion.

Reply:
(The GRN plot is generated by default if --plot = true with the influence function)

Reply:
oops!